docs/Project-Configuration/Parameterized-Builds.md

```diff
@@ -18,7 +18,7 @@ Parameters can be defined in multiple different places, and can overwrite existi
 
 * Project-specific default parameters defined in the user’s local configuration.
 
-* Default parameters defined for the patch alias on the Project Page, if applicable (only project admins have access to these). 
+* Default parameters defined for the patch alias on the Project Page, if applicable (only project admins have access to these). These do not apply to periodic builds. 
```

Comment:
Patch parameters will not apply to periodic builds in some of the above as well, (patch submission page, command line, etc). When part of one of five bullet points, people might make the assumption that it's only in this one that it doesn't apply. 
If we want to specifically specify it for the project page section, we can add it under "Project Page Usage". It already says " This is only available for CLI patches, not for PR patches." we can add to it to exclude periodic builds as well, maybe more general (and not available for non-patch builds including....")

Reply:
that works, thank you! 
It would be a good idea to add a note to that ticket to update these docs once it's implemented.